flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/LocalInputChannel.java

```diff
@@ -95,7 +96,8 @@ public LocalInputChannel(
 	// ------------------------------------------------------------------------
 
 	@Override
-	void requestSubpartition(int subpartitionIndex) throws IOException, InterruptedException {
+	@VisibleForTesting
```

Comment:
I guess it is not necessary to add this annotation which only indicates for `private` change.